### PR TITLE
Added Scribus icon #8722

### LINKED ISSUE #8722: Add scribus icon
### Brand Name

Scribus

### Website

https://www.scribus.net/

### Popularity Metric

258,258

### Official Resources for Icon and Color

https://wiki.scribus.net/canvas/Scribus_Desktop_Icons
https://de.wikipedia.org/wiki/Datei:Scribus_logo.svg

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PixelWelt**
close #8722

### CODE REVIEW

**Review by PixelWelt — COMMENTED** [review]
really fixed alphabetical order

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
The project only accepts single line SVG optimized to follow a uniform format. The icons should also be monochromatic. Please have a look at https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md and give it another go. Since the source icon has a lot of details and colors, some modification will need to be done to make it look good in monochromatic use.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11033,6 +11033,11 @@
             "source": "https://brand.scribd.com/d/oFZcsq7FVpSh/about-scribd#/media-press-kit/downloads",
             "guidelines": "https://brand.scribd.com/d/oFZcsq7FVpSh/about-scribd#/marketing-visual-guidelines/our-logo/our-logomark"
         },
+        {
+            "title": "Scribus",
+            "hex": "165885",
+            "source": "https://commons.wikimedia.org/wiki/File:Scribus_logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:11039]:
```suggestion
            "source": "https://wiki.scribus.net/canvas/Promotion_material"
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
I've found an official source through that original Wikipedia source. Would you be able to regenerate following [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) and using the SVG found there?

**Comment by PeterShaggyNoble** [comment]
There's now an [SVG](http://linux.hanski.info/static/scribus_promotion/scribus-icon2.svg) provided at [the original source](https://wiki.scribus.net/canvas/Promotion_material) so we should use that. We'll need to decide on a monochrome treatment first, though. One possible, and _very_ rough, option would be:

Scribus

**Comment by PeterShaggyNoble** [comment]
Closing due to lack of activity. @PixelWelt, if you would like to continue working on this then we can re-open this PR for you. If anyone else would like to take it on then please open a new PR.

**Pull request closed without merging**